### PR TITLE
lxde-base/lxdm: fix bug #597260 and clean up orphaned patches

### PR DESCRIPTION
**Pull request by stefson**
The lack of execinfo.h leads to a compile error of lxdm on all musl profiles, which was fixed by a patch that has been accepted by upstream. 

https://git.lxde.org/gitweb/?p=lxde/lxdm.git;a=commitdiff;h=855c090f1330aedc97b9a486dcd0d0b4c7ff4f4e;hp=72812894cfd9454d70e4b0753531e46580416771

Also I cleaned up all patches which were not referenced in any of the ebuilds.

### CODE REVIEW

File: lxde-base/lxdm/lxdm-0.4.1-r10.ebuild [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="2"
```
Comment (doughdemon) [inline lxde-base/lxdm/lxdm-0.4.1-r10.ebuild]: Please update the ebuild to EAPI 6, and run `repoman`.

File: lxde-base/lxdm/lxdm-0.5.3-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2016 Gentoo Foundation
```
Comment (doughdemon) [inline lxde-base/lxdm/lxdm-0.5.3-r1.ebuild]: no revision bump is necessary here. The ebuild is unstable, and the added patch only fixes a build failure.

File: lxde-base/lxdm/lxdm-0.4.1-r10.ebuild (hunk 1 above)
Comment (stefson) [inline lxde-base/lxdm/lxdm-0.4.1-r10.ebuild]: I'm sorry, but I don't know much about EAPI changes and everything related to that. Therefore someone else has to take care of this.

Comment (doughdemon) [inline lxde-base/lxdm/lxdm-0.4.1-r10.ebuild]: Do you specifically need this version? If not, I would not patch this version, wait for 0.5.3-r1 to get stabilized, and this version to get cleaned up.

Comment (stefson) [inline lxde-base/lxdm/lxdm-0.4.1-r10.ebuild]: Well, 0.4.1 is stable and I couldn't find any stable request for 0.5.3, therefore I'd state that someone running a system with stable keywords needs it. Both are broken with gtk-3.20, but that is another story.

**Comment by doughdemon** [comment]
Version 0.4.1-r1 will first be unstable and will therefore not help
stable users until it gets stabilized. Unless there is a reason for
keeping the old version, it seems easier to get the new version
patched, and stabilized after a while. (Until then it would be a
good idea to have the patch in the musl overlay.)

**Comment by stefson** [comment]
I created a new pull request at https://github.com/gentoo/gentoo/pull/2690

**Pull request closed without merging**